server_src/player.js

```diff
@@ -1050,10 +1051,8 @@ class Player {
     }
 
     EMP (t) {
- 	    if (this.ship >= 16 && this.ship <= 20) t *= 1.25; // Emp works better on elite ships.
-        if (this.ship === 21 && this.health * 1.05 < this.maxHealth) this.health *= 1.05; // It will also heal the ship a very small bit.
-
-        if (this.shield) t *= 0.33; // Shield offers some protection for electronic components
```

Comment:
Why did this go away? it's not documented in the change

Reply:
No idea, it was part of the EMP features that made Elite Ships weaker to that weapon, plus the reason why r21 didn't have a C-Slot on the first place.

Reply:
this is fixed in lb-changes though

Reply:
Reverted back to original.